data/map.txt

```diff
@@ -22573,6 +25734,56 @@ system Seginus
 			distance 416
 			period 37.7101
 
+system Sejuka
+	pos -71.791722 -153.812
+	government Uninhabited
+	attributes korath
+	asteroids "small rock" 13 2.6163
+	asteroids "medium rock" 1 2.4871
+	asteroids "large rock" 2 3.0362
+	asteroids "large metal" 1 2.6809
+	minables tungsten 1 3.1570134
+	minables aluminum 1 3.0425517
+	haze _menu/haze-133
+	fleet "Large Quarg" 1000
+	habitable 1641.25
+	belt 1029
+	trade Clothing 322
+	trade Electronics 826
+	trade Equipment 365
+	trade Food 564
+	trade "Heavy Metals" 1027
+	trade Industrial 622
+	trade "Luxury Goods" 1029
+	trade Medical 484
+	trade Metal 536
+	trade Plastic 281
+	object
+		sprite star/k5
+		distance 166.50724
+		period 68.873863
+		offset 180
+	object
+		sprite star/giant
```

Comment:
The old giant is kept only for backwards compatibility, it's intended to be removed in the future.  Probably not a good idea to add them.

Can't say much better for the `-old` stars either, I'd rather rename `k5-old` to something like `m-subgiant` or `m5iv`, but that's further down the road if ever.

```suggestion
		sprite star/k5-old
```